editor/components/block-settings-menu/reusable-block-toggle.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import { noop } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { IconButton } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { getBlock } from '../../selectors';
+import { convertBlockToStatic, convertBlockToReusable } from '../../actions';
+
+export function ReusableBlockToggle( { block, convertToStatic, convertToReusable } ) {
+	const isReusableBlock = block.name === 'core/reusable-block';
+
+	return (
+		<IconButton
+			className="editor-block-settings-menu__control"
+			icon="controls-repeat"
+			onClick={ isReusableBlock ? convertToStatic : convertToReusable }
+		>
+			{ isReusableBlock ? __( 'Detach from Reusable Block' ) : __( 'Make into a Reusable Block' ) }
```

Comment:
Would "Convert to Reusable Block" or "Convert to Global Block" be more concise?

Reply:
Changed in 619d81f51afbd3bf0fd5a407e2a82a993b7a7051.